core/translate/expr.rs

```diff
@@ -582,7 +584,21 @@ pub fn analyze_expr<'a>(expr: &'a Expr, column_info_out: &mut ColumnInfo<'a>) {
                 column_info_out.args = args;
             }
         }
-        ast::Expr::FunctionCallStar { .. } => todo!(),
+        ast::Expr::FunctionCallStar {
+            name,
+            filter_over: _,
+        } => {
+            let func_type =
+                match Func::resolve_function(normalize_ident(name.0.as_str()).as_str(), 1) {
+                    Ok(func) => Some(func),
+                    Err(_) => None,
+                };
+            if func_type.is_none() {
+                panic!("Function not found");
```

Comment:
Since the expr is a `FunctionCallStar`, should this be `unreachable`?

Reply:
If so, maybe this block should be 

```Rust
            if let Some(func) =
                Func::resolve_function(normalize_ident(name.0.as_str()).as_str(), 1).ok()
            {
                column_info_out.func = Some(func);
            } else {
                unreachable!();
            }
```

Reply:
Yes unreachable is perhaps better here